packages/plus/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -30,26 +110,51 @@ export const CodeEditor = ({
   onBlur,
   height,
   readOnly,
+  disabled = false,
+  label,
+  copyButton,
+  id,
+  helper,
+  labelDescription,
 }: CodeEditorProps) => (
-  <div>
-    {title ? (
+  <StyledStack gap={0.5} disabled={disabled}>
+    {label ? <Label labelDescription={labelDescription}>{label}</Label> : null}
+    {title && !label ? (
       <StyledText as="h3" variant="headingSmall">
         {title}
       </StyledText>
     ) : null}
```

Comment:
```suggestion
    {label || title ? <Label labelDescription={labelDescription}>{label ?? title}</Label> : null}
```
So this way we have the same style whatever prop is set